### PR TITLE
Update of error messages

### PR DESCRIPTION
**Pull request by TheoBoschetti**

### CODE REVIEW

**Review by BastienSaulnier — DISMISSED** [review]
SnE

File: backend/routes/ambassador.js [hunk 1]
```diff
@@ -182,16 +182,15 @@ router.patch("/:id", upload.single("img"), async (req, res) => {
                   return res
                     .status(500)
                     .send(
-                      "Error has occured during the attribution of the tag !" +
-                        err
+                      "Erreur lors de la modification de l'ambassadeur." + err
```
Comment (LoicBrassart) [inline backend/routes/ambassador.js]: For consistency's sake, don't send `err` there

File: backend/routes/association.js [hunk 2]
```diff
@@ -184,16 +182,15 @@ router.patch("/:id", upload.single("img"), async (req, res) => {
                   return res
                     .status(500)
                     .send(
-                      "Error has occured during the attribution of the tag !" +
-                        err
+                      "Erreur lors de la modification de l'association." + err
```
Comment (LoicBrassart) [inline backend/routes/association.js]: Same

File: backend/routes/auth.js [hunk 3]
```diff
@@ -33,12 +33,11 @@ router.post("/", (req, res) => {
 
 router.post("/login", (req, res) => {
   passport.authenticate("local", { session: false }, (err, user, info) => {
-    if (err)
-      return res.status(500).send("Error has occured during the connection !");
+    if (err) return res.status(500).send("Erreur lors de la connection.");
```
Comment (LoicBrassart) [inline backend/routes/auth.js]: "connexion" en Français

File: backend/routes/campaign.js [hunk 4]
```diff
@@ -154,11 +154,8 @@ router.post("/", upload.single("img"), (req, res) => {
           if (err)
             return res
               .status(500)
-              .send(
-                "Error has occured during the creation of the new campaign !" +
-                  err
-              );
-          return res.sendStatus(201);
+              .send("Erreur lors de la création de la campagne." + err);
```
Comment (LoicBrassart) [inline backend/routes/campaign.js]: Don't send `err`

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor suggestions

**Review by BastienSaulnier — APPROVED** [review]
SnE anymore ( sorry for my blindness about err display... )

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**